lib/matplotlib/font_manager.py

```diff
@@ -929,6 +940,52 @@ def set_fontconfig_pattern(self, pattern):
             else:
                 getattr(self, "set_" + key)(val)
 
+    @property
+    def mathtext_fontset(self):
+        """
+        Returns the current family of math fonts being used for this object.
+        If no font is set, returns the global font defined in 
+        :rc:`mathtext.fontset`.
+
+        Returns
+        -------
+        str: the name of the current fontset or rcParams['mathtext.fontset']
+        """
+
+        if self._mathtext_fontset is None:
+            return rcParams['mathtext.fontset']
+        return self._mathtext_fontset
+
+    @mathtext_fontset.setter
+    def mathtext_fontset(self, fontset):
+        """
+        The mathtext_fontset is the family of fonts used in case this text is
+        rendered in math mode. If no mathtext_fontset is defined, the
+        fontset defined in :rc:`mathtext.fontset` will be used.
+
+        Parameters
+        ----------
+        fontset : str, the name of the fontset. Available fontsets are defined
+        in the matplotlibrc.template file :ref:`here
+        <customizing-with-matplotlibrc-files>`
+
+        See Also
+        --------
+        .text.Text.set_mathtext_fontset
+        """
+        if fontset is None or fontset == '':
```

Comment:
The reason to support *fontset=None* is to make it more like the other *rcParams* (and to avoid some bugs, for instance, if I did this:
```
plt.rcParams['mathtext.fontset'] = 'dejavusans'
plt.text('text1')
plt.rcParams['mathtext.fontset'] = 'dejavuserif'
plt.text('text2')
```
The texts would have different fonts. I actually liked that change but the initial behavior was not like that. Also, the the interaction with functions like *ax.title()* and *ax.xlabel()* would not be correct because the Text object is created before these calls, I would probably need to change *ax.title()*, *ax.xlabel()*, *ax.ylabel()* and probably many others.
By leaving *mathtext_fontset = None* I ensure the original behavior is maintained, that is: the texts with no *math_font* will all be rendered with the *rcParam* present at the moment *parse()* is called.